app/main.py

```diff
@@ -1 +1,24 @@
-# write your code here
+import os
+
+
+def copy_file(command: str) -> None:
+    done_lst_path = command.split()
```

Comment:
Code Style Issue: The variable name `done_lst_path` is not descriptive enough. Consider renaming it to something more meaningful, such as `command_parts`.